packages/docs/src/routes/docs/cheat/qwik-react/playground_urls.tsx

```diff
@@ -0,0 +1,39 @@
+
+  clockExampleLink:
+    '/playground#files=JTVCJTdCJTIycGF0aCUyMiUzQSUyMiUyRmFwcC50c3glMjIlMkMlMjJjb2RlJTIyJTNBJTIyaW1wb3J0JTIwJTdCJTIwY29tcG9uZW50JTI0JTJDJTIwdXNlU3RvcmUlMkMlMjB1c2VDbGllbnRFZmZlY3QlMjQlMjAlN0QlMjBmcm9tJTIwJyU0MGJ1aWxkZXIuaW8lMkZxd2lrJyUzQiU1Q24lNUNuZXhwb3J0JTIwY29uc3QlMjBBcHAlMjAlM0QlMjBjb21wb25lbnQlMjQoKCklMjAlM0QlM0UlMjAlN0IlNUNuJTIwJTIwcmV0dXJuJTIwJTNDQ2xvY2slMjAlMkYlM0UlM0IlNUNuJTdEKSUzQiU1Q24lNUNuZXhwb3J0JTIwY29uc3QlMjBDbG9jayUyMCUzRCUyMGNvbXBvbmVudCUyNCgoKSUyMCUzRCUzRSUyMCU3QiU1Q24lMjAlMjBjb25zdCUyMHN0YXRlJTIwJTNEJTIwdXNlU3RvcmUoJTdCJTVDbiUyMCUyMCUyMCUyMHNlY29uZHMlM0ElMjAwJTJDJTVDbiUyMCUyMCU3RCklM0IlNUNuJTIwJTIwdXNlQ2xpZW50RWZmZWN0JTI0KCgpJTIwJTNEJTNFJTIwJTdCJTVDbiUyMCUyMCUyMCUyMGNvbnN0JTIwaW50ZXJ2YWwlMjAlM0QlMjBzZXRJbnRlcnZhbCgoKSUyMCUzRCUzRSUyMCU3QiU1Q24lMjAlMjAlMjAlMjAlMjAlMjBzdGF0ZS5zZWNvbmRzJTJCJTJCJTNCJTVDbiUyMCUyMCUyMCUyMCU3RCUyQyUyMDEwMDApJTNCJTVDbiUyMCUyMCUyMCUyMHJldHVybiUyMCgpJTIwJTNEJTNFJTIwY2xlYXJJbnRlcnZhbChpbnRlcnZhbCklM0IlNUNuJTIwJTIwJTdEKSUzQiU1Q24lNUNuJTIwJTIwcmV0dXJuJTIwJTNDZGl2JTNFU2Vjb25kcyUzQSUyMCU3QnN0YXRlLnNlY29uZHMlN0QlM0MlMkZkaXYlM0UlM0IlNUNuJTdEKSUzQiUyMiU3RCUyQyU3QiUyMnBhdGglMjIlM0ElMjIlMkZlbnRyeS5zZXJ2ZXIudHN4JTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMHJlbmRlclRvU3RyaW5nJTJDJTIwUmVuZGVyT3B0aW9ucyUyMCU3RCUyMGZyb20lMjAnJTQwYnVpbGRlci5pbyUyRnF3aWslMkZzZXJ2ZXInJTNCJTVDbmltcG9ydCUyMCU3QiUyMFJvb3QlMjAlN0QlMjBmcm9tJTIwJy4lMkZyb290JyUzQiU1Q24lNUNuZXhwb3J0JTIwZGVmYXVsdCUyMGZ1bmN0aW9uJTIwKG9wdHMlM0ElMjBSZW5kZXJPcHRpb25zKSUyMCU3QiU1Q24lMjAlMjByZXR1cm4lMjByZW5kZXJUb1N0cmluZyglM0NSb290JTIwJTJGJTNFJTJDJTIwb3B0cyklM0IlNUNuJTdEJTVDbiUyMiU3RCUyQyU3QiUyMnBhdGglMjIlM0ElMjIlMkZyb290LnRzeCUyMiUyQyUyMmNvZGUlMjIlM0ElMjJpbXBvcnQlMjAlN0IlMjBBcHAlMjAlN0QlMjBmcm9tJTIwJy4lMkZhcHAnJTNCJTVDbiU1Q25leHBvcnQlMjBjb25zdCUyMFJvb3QlMjAlM0QlMjAoKSUyMCUzRCUzRSUyMCU3QiU1Q24lMjAlMjByZXR1cm4lMjAoJTVDbiUyMCUyMCUyMCUyMCUzQ2h0bWwlM0UlNUNuJTIwJTIwJTIwJTIwJTIwJTIwJTNDaGVhZCUzRSU1Q24lMjAlMjAlMjAlMjAlMjAlMjAlMjAlMjAlM0N0aXRsZSUzRUhlbGxvJTIwUXdpayUzQyUyRnRpdGxlJTNFJTVDbiUyMCUyMCUyMCUyMCUyMCUyMCUzQyUyRmhlYWQlM0UlNUNuJTIwJTIwJTIwJTIwJTIwJTIwJTNDYm9keSUzRSU1Q24lMjAlMjAlMjAlMjAlMjAlMjAlMjAlMjAlM0NBcHAlMjAlMkYlM0UlNUNuJTIwJTIwJTIwJTIwJTIwJTIwJTNDJTJGYm9keSUzRSU1Q24lMjAlMjAlMjAlMjAlM0MlMkZodG1sJTNFJTVDbiUyMCUyMCklM0IlNUNuJTdEJTNCJTVDbiUyMiU3RCU1RA%3D%3D',
```

Comment:
this example uses `useClientEffect` which is marked as deprecated now. maybe we should adjust that?